packages/server/src/fhir/routes.ts

```diff
@@ -256,6 +257,9 @@ function initInternalFhirRouter(): FhirRouter {
   router.add('GET', '/Patient/:id/$summary', patientSummaryHandler);
   router.add('POST', '/Patient/:id/$summary', patientSummaryHandler);
 
+  // Patient $update-account operation
+  router.add('POST', '/Patient/:id/$update-accounts', patientUpdateAccountHandler);
```

Comment:
There's inconsistent use in the docs between `$update-account` and `$update-accounts`: should standardize presumably on the second one.  Also this name makes it sound like the Patient is being updated; I'd suggest perhaps e.g. `$propagate-accounts` for clarity